src/applications/claims-status/components/AddFilesForm.jsx

```diff
@@ -283,28 +283,20 @@ class AddFilesForm extends React.Component {
             </div>
           ),
         )}
-        <Checkbox
-          onValueChange={checked => {
-            this.setState({ checked });
+        <VaCheckbox
+          onVaChange={event => {
+            this.setState({ checked: event.detail.checked });
           }}
           checked={this.state.checked}
-          errorMessage={this.state.errorMessageCheckbox}
-          label={
-            // eslint-disable-next-line react/jsx-wrap-multilines
-            <div>
-              <strong>
-                The files I uploaded are supporting documents for this claim
-                only.
-              </strong>
-              <div className="vads-u-padding-top--1">
-                To submit supporting documents for a new disability claim,
-                please visit our{' '}
-                <a href="/disability/how-to-file-claim">How to File a Claim</a>{' '}
-                page.
-              </div>
-            </div>
-          }
+          error={this.state.errorMessageCheckbox}
+          message-aria-describedby="To submit supporting documents for a new disability claim, please please visit our How to File a Claim page."
```

Comment:
There are 2 pleases here and in the div below

Reply:
Hi @jerekshoe Thanks for taking a look. I'm not sure I understand your comment here though. Is there something that needs to be updated?

Reply:
Oh, I see! There's a typo "**please please** visit our How to File a Claim page."

It looks like this was in the original code so I didn't notice but I'll get it cleaned up now.